website/pages/en/cookbook/subgraph-debug-forking.mdx

```diff
@@ -44,12 +44,12 @@ export function handleUpdatedGravatar(event: UpdatedGravatar): void {
 }
 ```
 
-Oops, how unfortunate, when I deploy my perfect looking subgraph to the [hosted service](https://thegraph.com/hosted-service/) it fails with the _"Gravatar not found!"_ error.
+Oops, how unfortunate, when I deploy my perfect looking subgraph to [Subgraph Studio](https://thegraph.com/studio/) it fails with the _"Gravatar not found!"_ error.
 
 The usual way to attempt a fix is:
 
 1. Make a change in the mappings source, which you believe will solve the issue (while I know it won't).
-2. Re-deploy the subgraph to the [hosted service](https://thegraph.com/hosted-service/) (or another remote Graph node).
+2. Re-deploy the subgraph to [Subgraph Studio](https://thegraph.com/studio/) (or another remote Graph node).
```

Comment:
```suggestion
2. Re-deploy the subgraph to [Subgraph Studio](https://thegraph.com/studio/) (or another remote Graph Node).
```
Source: https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#0de2db33f11c4b3a970ed42cd97bc991